data/json/vehicle_parts.json

```diff
@@ -3601,7 +3601,7 @@
         "range" : 12,
         "fuel_type" : "plasma",
         "item" : "plasma_rifle",
-        "difficulty" : 7,
+        "difficulty" : 6,
```

Comment:
Exotic energy turrets seem like 7-8 territory: energy weapons were just getting out of the labs and into military/LEO first-rounds procurement & field use when the Cataclysm hit.  (They aren't prototypes/betas, but neither are they tried, true, and regular tech.  Lotsa unknowns yet to work out.)